packages/@aws-cdk/aws-apigateway/test/integ.stepfunctions-api-request-context-empty.ts

```diff
@@ -0,0 +1,42 @@
+import * as sfn from '@aws-cdk/aws-stepfunctions';
```

Comment:
We don't need so many integration tests. Integration tests are somewhat expensive (to maintain) than unit tests. We plan to, in the future, run these as part of a nightly build, so some frugality is expected.

The main thing that the integration test should be ensuring is that using `StepFunctionRestApi` correctly configures the integration between APIGateway and Step Functions. 
For other things like testing request context, use unit tests.

I think, just one integration test here would be sufficient.